core-common/src/main/java/org/mapstruct/IterableMapping.java

```diff
@@ -47,6 +47,14 @@
     String dateFormat() default "";
 
     /**
+     * A format string as processable by {@link java.text.DecimalFormat} if the annotated method maps from a
```

Comment:
Minor nit-picking: Can you import the type and use the short name here? That reads nicer in rendered docs.